setup.py

```diff
@@ -57,6 +54,15 @@
         'torchtext==0.12.0',
         'cnest',
     ],  # And any other dependencies alf needs
+    ext_modules=[
+        Pybind11Extension(
+            'environments._penv',
+            sources=['alf/environments/parallel_environment.cpp'],
+            extra_compile_args=[
+                '-O3 -Wall -std=c++17', '-fPIC', '-fvisibility=hidden'
+            ])
+    ],
+    cmdclass={'build_ext': build_ext},
     extras_require={
         'metadrive': ['metadrive-simulator==0.2.5.1', ],
     },
```

Comment:
You need to include the compiled so file in the 'package_data'? Otherwise an installed alf package will miss the so file.